src/FileIO.scala

```diff
@@ -0,0 +1,41 @@
+import java.io.{File, PrintWriter}
+
+import scala.collection.mutable
+import scala.io.Source
+
+/**
+  * Created by Zach on 10/18/16.
+  */
+object FileIO {
+
+  def main(args: Array[String]): Unit = {
+
+    val purchases = mutable.MutableList[(String, String, String, String, String)]()
+    def prompt(s: String) = {
+      println(s);
+      io.StdIn.readLine()
+    }
+
+    Source.fromFile("purchases.csv").getLines().drop(1).foreach(line => {
+      var purchase = {
+        val Array(customer_id, date, credit_card, cvv, category) = line.split(",").map(_.trim)
+        (customer_id, date, credit_card, cvv, category)
+      }
+      purchases += purchase
+    })
+
+    val userInput = prompt("What category would you like to see? [Furniture] [Alcohol] [Toiletries] [Shoes] [Food] [Jewelry]")
+    userInput match {
+      case s => {
```

Comment:
Re-write this as two cases; one with a test and one with the '_' wildcard. ie; case s if s.isEmpty => { .. }
and then, case _ => { .. }

So, when matched on a string:  create ONLY ONE text.foreach line that includes both display pring and file output.